.github/workflows/javascript.yml

```diff
@@ -17,8 +17,12 @@ jobs:
         module:
           - name: 'mysql2'
             with_oceanbase_container: true
+          - name: 'prisma'
+            with_oceanbase_container: true
+            oceanbase_image_tag: '4.2.2'
```

Comment:
only `oceanbase-ce >= 4.2.2` supports `utf8mb4_unicode_ci` collation and that is required for Prisma.